### ISSUE TITLE
Is there an example of how to use the 1.5 version with maven?

### ISSUE THREAD

**Issue by rmoskal** [camunda/camunda-bpm-reactor#52]
I had been using the camunda-bpm-reactor-spring-starter but there seems to be all sorts of issues with 7.6. For example my userTask listener works, but receive Task has stopped.

**Comment by jangalinski** [comment]
could you specify "all sorts of problems"?

**Comment by rmoskal** [comment]
I used to be able to listen to recieveTask(s) starting up like so:

```
@Component
@CamundaSelector(type = "receiveTask", event = ExecutionListener.EVENTNAME_START)
public class ReceiveTaskListener implements ExecutionListener {
...
```

That no longer works.

I can still get userTasks like so:

> @Component
> @CamundaSelector(type = "userTask", event = TaskListener.EVENTNAME_CREATE)
> public class UserTaskListener extends ReactorTaskListener {

I also trap things like "startEvent", even rewriting the receiveTask listener in this form doesn't work:

```
@Component
@CamundaSelector(type = "receiveTask", event = TaskListener.EVENTNAME_CREATE)
public class ReceiveTaskListener extends ReactorTaskListener {
```

I'm working with this version of the code:

      <dependency>
            <groupId>org.camunda.bpm.extension</groupId>
            <artifactId>camunda-bpm-reactor-spring-starter</artifactId>
            <version>1.2</version>
        </dependency>

and camunda 7.6

**Comment by jangalinski** [comment]
Hi Robert ... so you are still on 1.2, ok, the title suggets otherwise. 
Your sample code looks suspicious ... you are 

```
@Component
@CamundaSelector(type = "receiveTask", event = ExecutionListener.EVENTNAME_START)
public class ReceiveTaskListener implements ExecutionListener {
```

using an "ExecutionListener" here, not a ReactorExecutionListener, meaning you will have to register this component on the bus yourself .... your TaskLIstener uses a ReactorTaskListener which already contains an autowired method to do so ....  and in your last example, you register a tasklistener "create" on a receiveTask ... this will not work because taskListener can only be applied on user tasks (bpmn/cmmn) .... in fact, I wonder why this does not raise an exception/error-log ...
Please check if these are just typos ... if the problem remains, it would be perfect if you could provide a small example (minimal, working) so I can reproduce it. thanks. Jan

**Comment by rmoskal** [comment]
No you are right. I started mixing up the techniques from 1.5 with my current ones.  Sorry for the confusion and the taking up of your time. All is well now.

May I ask when 1.5 will be released?  Can one get access to the SNAPHOT releases?

**Comment by jangalinski** [comment]
I updated the readme with the snapshot coordinates https://github.com/camunda/camunda-bpm-reactor/blob/master/README.adoc#maven-coordinates

Regarding the release: I would like to fix #51 and #46 first. I cannot promise anything but it should not take that long ... in doubt, we should stick to the open source paradigma: publish early, publish often
